go.mod

```diff
@@ -1,22 +1,20 @@
 module github.com/moby/buildkit
 
-go 1.12
+go 1.11
```

Comment:
why is this `go1.11` ?

Reply:
The change to `go 1.12` must've been recent; I went over all the changes in the dependencies during rebase, however I missed the version metadata. I've updated it.